samples/models/extra-capabilities.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Model
+metadata:
+  name: custom-model
```

Comment:
:thought_balloon: Really minor, but this isn't a custom model in any sense, just one that adds extra caps

Reply:
Fixed this along with the filename in https://github.com/SeldonIO/seldon-core/pull/4718/commits/a7b4f0513dcc4e2e425ef5e1e5984438c3da43d9.